packages/x-components/package.json

```diff
@@ -30,7 +30,7 @@
     "./tests/**/*",
     "**/*.scss",
     "**/*.scss.js",
-    "**/*vue_styles.*.vue.js",
+    "**/*vue[0-9].js",
```

Comment:
Due to how rollup generates now the output filenames when preserving modules, we need to adjust the sideEffect field in the package.json so bundlers do not tree shake styles of vue components.